kafka-connector/src/main/java/com/google/pubsub/kafka/sink/CloudPubSubSinkTask.java

```diff
@@ -346,5 +350,21 @@ private void createPublisher() {
   }
 
   @Override
-  public void stop() {}
+  public void stop() {
+    log.info("Stop CloudPubSubSinkTask");
+
+    if (publisher != null) {
+      log.info("Shutting down PubSub publisher");
```

Comment:
shutting or shut (to be consistent with Stop CloudPubSubSinkTask)?

Reply:
I don't like `Stop` either but seemed consistent with `Start` in `start()` method.
I will change `Stop` to `Stopping`... and we'll check the reaction in official PR ;p